third_party/nvidia/backend/compiler.py

```diff
@@ -461,6 +461,12 @@ def make_cubin(self, src, metadata, opt, capability):
             # Accept more ptxas options if provided
             ptx_extra_options = opt.ptx_options.split(" ") if opt.ptx_options else []
 
+            # Accept more ptxas options if ptxas related envs are provided
+            if not knobs.nvidia.disable_ptxas_opt and (ptxas_options := os.environ.get("PTXAS_OPTIONS", None)):
```

Comment:
1. `PTXAS_OPTIONS` itself should be a knob
2. I'm not sure if we should override `ptx_extra_options` or append new options to the end. Overriding makes more sense  to me
3. `PTXAS_OPTIONS_KERNEL` is not necessary

With all these being said, if we want it to be a knob, its behavior should be simple enough.

Reply:
there is already `ptxas_options` knob to do the same thing, we shouldn't add a second way to do it

Reply:
I think that's reasonable